js/components/contactList/contactList_controller.js

```diff
@@ -67,22 +67,37 @@ angular.module('contactsApp')
 		});
 	});
 
+	// Get contacts
 	ContactService.getAll().then(function(contacts) {
 		$scope.$apply(function() {
 			ctrl.contacts = contacts;
-			// If desktop version, load first contact (see css for min-width media query)
-			if (!_.isEmpty(ctrl.contacts) && $(window).width() > 768) {
-				ctrl.setSelectedId(_.sortBy(contacts, function(contact) {
-					return contact.fullName();
-				})[0].uid());
+		});
+	});
+
+	// Wait for ctrl.contactList to be updated, load the first contact and kill the watch
```

Comment:
I'm planning to move this functionality to the angular routing (probably ui-router) in the future.
But good for now.